packages/backend/src/services/AsyncQueryService/AsyncQueryService.ts

```diff
@@ -286,10 +462,7 @@ export class AsyncQueryService extends ProjectService {
                 };
             case QueryHistoryStatus.PENDING:
                 if (resultsCacheEnabled && cacheKey && this.cacheService) {
-                    const cache = await this.cacheService.findCache(
-                        cacheKey,
-                        projectUuid,
-                    );
+                    const cache = await this.findCache(cacheKey, projectUuid);
```

Comment:
Replaced all the `this.cacheService.` with `this.` but kept the checks to the service and the `resultsCacheEnabled` to keep same business logic.